src/types.ts

```diff
@@ -85,3 +86,27 @@ export interface HomeAssistant {
   sendWS: (msg: MessageBase) => Promise<void>;
   callWS: <T>(msg: MessageBase) => Promise<T>;
 }
+
+export type ClimateEntity = HassEntity & {
```

Comment:
This is kinda annoying as HassEntity also defines `attributes = {[s: string]: any}`, which means autocomplete doesn't work. I opened https://github.com/home-assistant/home-assistant-js-websocket/pull/64 to address that. Will update this in a future PR.